package.json

```diff
@@ -1,21 +1,57 @@
 {
-  "name": "bb-practicum-team-1-back",
+  "name": "goal-tracker",
```

Comment:
goal-tracker-api ? it isn't a big deal though

Reply:
Fixed!